pages/models.py

```diff
@@ -2938,4 +2939,50 @@ class MathQuizPage(Page):
         APIField('search_description'),
     ]
 
-    parent_page_type = ['pages.HomePage']
\ No newline at end of file
+    parent_page_type = ['pages.HomePage']
+
+
+class LLPHPage(Page):
+    heading = models.CharField(max_length=255)
```

Comment:
We need a hero background image (statue of liberty in the design)

Reply:
Done